Added support for enable/disable organization checkout

mbooh opened the pull request:
Added support for enable/disable organization checkout
Settings in Commerce Manager to enable organization checkout

marisks commented:
Please fix conflicts.

mbooh commented:
Can I do that? I’m not really sure how that is done.
Ok, I will look into it.

Skickat från min iPhone

23 nov. 2017 kl. 08:37 skrev Māris Krivtežs <notifications@github.com<mailto:notifications@github.com>>:


Please fix conflicts.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Geta/EPi.Commerce.Payments/pull/43#issuecomment-346547447>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AQKH8q8o4WwN8MWUsENtOlNTjq93XJe3ks5s5SClgaJpZM4QnhPY>.

marisks commented:
You have to pull the changes from the main repository. Then it will ask you to merge.

mbooh commented:
I haved pulled the master branch and then merged it into my bransch but I get no conflicts? What am I missing?

marisks commented:
Probably it merged automatically. Push the changes to GitHub and it should be fine.

mbooh commented:
Strange, eveything is pushed. Is there still a conflict?

marisks commented:
Yes, there are still conflicts. Have you pushed the changes into enable_organization_checkout branch?
Have you merged master from the main repo into  enable_organization_checkout?

mbooh commented:
I have checked out the master bransch, created a branch called enable_organization_checkout from the master branch. Made my changes, tried to merge master into enable_organization_checkout again and that worked fine without conflicts. Pushed my enable_organization_checkout branch, also worked just fine. I guess that is the way do do?

marisks commented:
But have you pulled the latest master and then merged it into enable_organization_checkout?
Something like this:

```
git checkout master
git pull Geta master
git checkout enable_organization_checkout
git merge master
git push
```

mbooh commented:
Yes, that is done but using SourceTree. Now I just tried with the command tool with this error:

$ git pull Geta master
fatal: 'Geta' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

marisks commented:
It could be called "origin" too.
You can find remote names by:
```
git remote
```

mbooh commented:
Hmm, I can't get this to work.
$ git pull origin master
From https://github.com/mbooh/EPi.Commerce.Payments
 * branch            master     -> FETCH_HEAD
Already up-to-date.

is this correct? I tried to checkout the master branch in a new folder and create a new branch there and it worked fine and I got the conflicts but then I can't push to my branch

git -c diff.mnemonicprefix=false -c core.quotepath=false push -v --tags --set-upstream origin organizationcheckout:enable_organization_checkout
remote: Permission to Geta/EPi.Commerce.Payments.git denied to mbooh.
fatal: unable to access 'https://github.com/Geta/EPi.Commerce.Payments.git/': The requested URL returned error: 403

Pushing to https://github.com/Geta/EPi.Commerce.Payments.git

Annoying!

marisks commented:
As I see you have completely different master here:
https://github.com/mbooh/EPi.Commerce.Payments/commits/master

But master on the main repo looks like this:
https://github.com/Geta/EPi.Commerce.Payments/commits/master

Once, you get those same, you could merge the master into **enable_organization_checkout**

mbooh commented:
Yes. I manged to checkout the “real” master and the I merged it into my branch so far so good. But now I can’t push it?

git -c diff.mnemonicprefix=false -c core.quotepath=false push -v --tags origin enable_organization_checkout:enable_organization_checkout
remote: Permission to Geta/EPi.Commerce.Payments.git denied to mbooh.
fatal: unable to access 'https://github.com/Geta/EPi.Commerce.Payments.git/': The requested URL returned error: 403

Pushing to https://github.com/Geta/EPi.Commerce.Payments.git
Completed with errors, see above.

mbooh commented:
Ok, I changed the url i SourceTree to https://github.com/Geta/EPi.Commerce.Payments.git/
Checked out the master branch
Merged it with my branch
Changed the url back to https://github.com/mbooh/EPi.Commerce.Payments.git/
And then made the push without a 403 exception

Now I Think it should be fine.

mbooh commented:
That was strange. The file is now updated.

marisks commented on Geta.EPi.Commerce.Payments.Klarna.Checkout/ChangeLog.txt:
```diff
@@ -1,3 +1,6 @@
+20/11/2017 - Version 1.1.7.9000
```
The version number is same as the previous one.

marisks commented on Geta.EPi.Commerce.Payments.Klarna.Checkout/Geta.Epi.Payments.Klarna.CommerceManager/Apps/Order/Payments/Plugins/KlarnaCheckout/ConfigurePayment.ascx.cs:
```diff
@@ -42,19 +41,17 @@ public void LoadObject(object dto)
         public void SaveChanges(object dto)
         {
             if (!Visible)
-            { 
```
Keep brackets or have a return statement on the same line as the "if".

marisks commented on Geta.EPi.Commerce.Payments.Klarna.Checkout/Geta.Epi.Payments.Klarna.CommerceManager/Apps/Order/Payments/Plugins/KlarnaCheckout/ConfigurePayment.ascx.cs:
```diff
@@ -42,19 +41,17 @@ public void LoadObject(object dto)
 
             var paymentMethod = dto as PaymentMethodDto;
             if (paymentMethod == null)
-            { 
```
Keep brackets or have a return statement on the same line as the "if".
